sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering-validation.models.ts

```diff
@@ -614,6 +642,188 @@ export class OfferingValidationModel {
       "Not able to find a program related to this offering or it was not provided.",
   })
   programContext: EducationProgramForOfferingValidationContext;
+
```

Comment:
Please follow the class patterns, removing the blank lines between the properties.